sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -280,6 +280,15 @@
         return new Bound<>(name, filepattern, type, schema, false);
       }
 
+      /**
+       * Configures the source to succeed if the specified {@code fileOrPatternSpec} does not match any
+       * files. The default behavior is to fail on empty input.
+       */
+      public Bound<T> withAllowedEmptyInput() {
```

Comment:
Feel free to bikeshed on the API name. I think it should be consistent across all transform builders where we provide it.